src/routes/api.js

```diff
@@ -1,6 +1,25 @@
 const express = require("express");
+const passport = require('passport');
 const router = express.Router();
-const { businesses, businessesId, newReview } = require("../controllers/api");
+const { businesses, businessesId, newReview, googleOAuth, facebookOAuth, } = require("../controllers/api");
+const passportGoogle = passport.authenticate('googleToken', { session: false });
+
+// auth login
+router.get('/auth/login', (req, res) => {
+    res.send(req.user);
+});
+
+// auth logout
+router.get('/auth/logout', (req, res) => {
+    // handle with passport
+    req.logout();
+    //res.redirect('/');
+});
+
+
+//google & facebook oauth
+router.post('/oauth/google', passportGoogle, googleOAuth);
+router.post('/oauth/facebook', passport.authenticate('facebookToken', { session: false }), facebookOAuth);
```

Comment:
why session false here, while you are using session

Reply:
no session anymore, use in cookie parser